# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kbd/devfs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./openvt/openvt.h.orig	2002-10-11 14:08:59.000000000 +0300
+++ ./openvt/openvt.h	2003-08-16 15:48:07.000000000 +0300
@@ -33,8 +33,8 @@
  * Where your VTs are hidden
  */
 #ifdef __linux__
-#define VTNAME "/dev/tty%d"
-#define VTNAME2 "/dev/vc/%d"
+#define VTNAME "/dev/vc/%d"
+#define VTNAME2 "/dev/tty%d"
 #endif
 
 #ifdef ESIX_5_3_2_D
